resources/profiles/Zonestar.ini

```diff
@@ -383,289 +681,514 @@ machine_max_jerk_y = 10
-toolchange_gcode = G92 E0\nG1 E-30 F1200;pull filament\nG92 E0\nT[next_extruder];Change extruder\nG92 E0\nG1 E15 F1800; push filament-1\nG1 E25 F900; pushfilament-2\nG1 E29 F1200; push filament-3\nG92 E0
-use_relative_e_distances = 1
-wipe = 0,0,0,0
-layer_gcode = G92 E0
+travel_speed = 150
+machine_max_acceleration_e = 10000
+machine_max_acceleration_extruding = 1500
+machine_max_acceleration_retracting = 1500
+machine_max_acceleration_travel = 1500
+machine_max_acceleration_x = 9000
+machine_max_acceleration_y = 9000
+machine_max_acceleration_z = 500
+machine_max_feedrate_e = 120
+machine_max_feedrate_x = 500
+machine_max_feedrate_y = 500
+machine_max_feedrate_z = 12
+machine_max_jerk_e = 2.5
+machine_max_jerk_x = 10
+machine_max_jerk_y = 10
+machine_max_jerk_z = 0.2
+machine_min_extruding_rate = 0
+machine_min_travel_rate = 0
 
-[printer:Zonestar Z8+DDE]
+## common for Z9MAX
+[printer:*Z9Max*]
 inherits = *common*
-printer_model = Z8
-printer_variant = 0.4
-bed_shape = 0x0,310x0,310x310,0x310
-default_filament_profile = Generic TPU @Zonestar
-default_print_profile = 0.20mm @Zonestar SINGLE
-deretract_speed = 30
-end_gcode = G28 X\nM106 S0 ; turn off FAN\nM104 S0 ; turn off extruder\nM140 S0 ; turn off bed\nM84 ; disable motors
-extruder_colour = #00FF00
-extruder_offset = 0x0
-max_layer_height = 0.3
+printer_model = Z9Max
+bed_shape = 0x0,500x0,500x500,0x500
 max_print_height = 400
-min_layer_height = 0.1
+travel_speed = 120
+machine_max_acceleration_e = 10000
+machine_max_acceleration_extruding = 1500
+machine_max_acceleration_retracting = 1500
+machine_max_acceleration_travel = 1500
+machine_max_acceleration_x = 9000
+machine_max_acceleration_y = 9000
+machine_max_acceleration_z = 500
+machine_max_feedrate_e = 120
+machine_max_feedrate_x = 500
+machine_max_feedrate_y = 500
+machine_max_feedrate_z = 12
+machine_max_jerk_e = 2.5
+machine_max_jerk_x = 10
+machine_max_jerk_y = 10
+machine_max_jerk_z = 0.2
+machine_min_extruding_rate = 0
+machine_min_travel_rate = 0
+
+#================================================================================
+#  End of "printer>>Common"
+#================================================================================
+
+#================================================================================
+#  printer>>Extruders
+#================================================================================
+## Extruders >> Bowden standart Extruder
+[printer:*Bowden*]
+deretract_speed = 30
+extruder_colour = #FF0000
+extruder_offset = 0
+retract_before_travel = 8
+retract_before_wipe = 0%
+retract_layer_change = 0
+retract_length = 10
+retract_length_toolchange = 15
+retract_lift_above = 0
+retract_lift_below = 0
+retract_restart_extra = 0
+retract_restart_extra_toolchange = 0.2
+retract_speed = 40
+use_relative_e_distances = 0
+wipe = 0
+max_volumetric_speed = 5
```

Comment:
This key does not belong to printer section. Remove it from all printer sections.

You can run prusaslicer with --loglevel=5 parameter to see all errors.